platform/jewel/foundation/src/main/kotlin/org/jetbrains/jewel/foundation/lazy/tree/KeyActions.kt

```diff
@@ -223,17 +228,23 @@ public class DefaultTreeViewKeyActions(
         state: SelectableLazyListState,
         selectionMode: SelectionMode,
     ): KeyEvent.() -> Boolean = lambda@{
+        // Explicitly don't handle Tab key events - let them pass through for focus traversal
+        if (key == Key.Tab) return@lambda false
+
```

Comment:
Regarding the Tab key behavior: I see that I still have to press Tab multiple times to move focus to the next component after the tree. The tree should be a single tab stop, for example, like the tree in the sidebar in the UI Sandbox dialog in the IDE. Is this behavior expected for now?